docs/user/TerraformAzureAuthentication.md

```diff
@@ -30,24 +65,29 @@ source $HOME/.azure_creds.sh
 
 This will pull in those values into your current terminal session. Any terraform commands submitted in that session will use those values.
 
-## Docker 
+### Set Authentication Variables when using the Docker container
 
-Run these commands to initialize the environment for the project. These commands will need to be run and pulled  into your environment each time you start a new session to use this repo and terraform.
+When using the docker container to run terraform, ru these commands to initialize the environment for the project. These commands will need to be run and pulled into your environment each time you start a new terminal session.
```

Comment:
typos - "..Docker container to run Terraform, run.."
Use 'initialized or setup" instead of "pulled into"

Reply:
reworded the docker instructions.

Reply:
Small typo here, ru -> run ;)

Reply:
check out my latest commit. I reworded this section